### PR TITLE
Chore telemetry autologin with 2fa

### PR DESCRIPTION
**Pull request by Karim94**
#### Relevant info and issue/PR links 
https://app.clubhouse.io/peerio/story/14137/mobile-add-telemetry-for-autologin-with-and-without-2fa
https://github.com/PeerioTechnologies/peerio-icebear/pull/336

#### Testing instructions  
Added telemetry for login success and failure with options for 2SV on/off and manual/auto login.

----
### Repository owner

- [ ] Was tested and can be merged.
- [ ] PR name follows conventional changelog format.

[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)

### CODE REVIEW

File: app/components/settings/two-factor-auth.js [hunk 1]
```diff
@@ -58,10 +60,18 @@ async function twoFactorAuthPopup(active2FARequest) {
     }
     const { value, checked } = result;
     uiState.trustDevice2FA = checked;
-    submit(value, checked);
+    try {
+        await submit(value, checked);
+    } catch (e) {
+        console.error(e);
+        tm.login.onUserTfaLoginFailed(User.current.autologinEnabled);
```
Comment (seavan) [inline app/components/settings/two-factor-auth.js]: this line will be executed for any kind of 2fa error, not just login. is it an expected behaviour?

Comment (Karim94) [reply app/components/settings/two-factor-auth.js]: No it isn't expected, changing it to only send that error when 2fa fails on login.

**Comment by ebilg** [comment]
can be merged

**Comment by Karim94** [comment]
@seavan fixed

**Pull request merged**